.github/workflows/pdebuild.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   build:
-
+    name: Ubuntu pdebuild on Ubuntu
```

Comment:
TYPO?,  Pdebuild on Ubuntu ?

Reply:
creating Ubuntu packages with pdebuild on Ubuntu VM. :)
(you can create Debian packages with pdebuild, too)